modules/rosa-hcp-aws-privatelink-create-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-aws-privatelink-create-cluster_{context}"]
+= Creating an AWS PrivateLink cluster
+
+You can create a private cluster with multiple availability zones (Multi-AZ) on {hcp-title} using the {product-title} (ROSA) CLI, `rosa`.
+
+[NOTE]
+====
+AWS PrivateLink is supported on existing VPCs only.
+====
+
+.Prerequisites
+
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest {product-title} (ROSA) CLI, `rosa`, on your installation host.
+* You have created a VPC with at least one private subnet.
+
+.Procedure
+
+Creating a cluster with {hcp} can take around 10 minutes.
+
+. Create a virtual private cloud. Ensure that your machine's classless inter-domain routing (CIDR) match your virtual private cloud's CIDR. See https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc[Requirements for using your own VPC] and link:https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-validation_installing-aws-vpc[VPC Validation] for more information.
```

Comment:
```suggestion
. Create a VPC. Ensure that your machine's classless inter-domain routing (CIDR) matches your virtual private cloud's CIDR. See https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc[Requirements for using your own VPC] and link:https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-validation_installing-aws-vpc[VPC Validation] for more information.
```
This step is a little confusing given the prerequisites. The last prereq was that you have created a VPC with at least one subnet, and now the first step is to create a VPC with at least one subnet. Do they need two VPCs?